tests/test_url.py

```diff
@@ -58,6 +58,20 @@ def test_safe_url_string(self):
         self.assertEqual(safeurl, "http://www.example.com/%A3")
 
         self.assertTrue(isinstance(safe_url_string(b'http://example.com/'), str))
+        
+        # Remove ASCII tab and newlines
```

Comment:
Could you please extract these tests to a separate test method?

Reply:
Done, sorry I missed that, thanks.